CHANGELOG.md

```diff
@@ -1,7 +1,14 @@
-# dbt_shopify_source v0.UPDATE.UPDATE
+# dbt_shopify_source v0.8.3
 
- ## Under the Hood:
+## Bug Fixes
+Here are the fixes introduced in [PR #69](https://github.com/fivetran/dbt_shopify_source/pull/69):
```

Comment:
Very minor, but I think typically when we list multiple changes for 1 PR, we lead with the PR 

So "PR #69 includes the following fixes:"

Reply:
Changed around!